package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@ideapod/hapi-aws-sqs",
+  "version": "0.0.1",
```

Comment:
Use `0.0.0` as the initial version. Version number gets changed as part of the release process.